stdlib/private/SwiftReflectionTest/CMakeLists.txt

```diff
@@ -3,14 +3,17 @@ if (SWIFT_INCLUDE_TESTS)
   add_swift_library(swiftSwiftReflectionTest ${SWIFT_STDLIB_LIBRARY_BUILD_TYPES} IS_STDLIB
     SwiftReflectionTest.swift
     SWIFT_MODULE_DEPENDS Darwin
+    TARGET_SDKS ALL_APPLE_PLATFORMS
     INSTALL_IN_COMPONENT stdlib-experimental)
 
   foreach(SDK ${SWIFT_SDKS})
-    foreach(ARCH ${SWIFT_SDK_${SDK}_ARCHITECTURES})
-      set(VARIANT_SUFFIX "-${SWIFT_SDK_${SDK}_LIB_SUBDIR}-${ARCH}")
-      add_dependencies(
-        "swiftSwiftReflectionTest${VARIANT_SUFFIX}"
-        "swift-reflection-test${VARIANT_SUFFIX}")
-    endforeach()
+    if("${SDK}" IN_LIST SWIFT_APPLE_PLATFORMS)
```

Comment:
Apologies, looks like I lost a line of source code when I split up #4972. I amended the commit with the missing CMake changes that add `SWIFT_APPLE_PLATFORMS`. 😨